### ISSUE TITLE
Secondary info is too small/hard to read on desktop

### ISSUE THREAD

**Issue by Xlinx64** [selvalt7/modern-circular-gauge#17]
### Issue
The secondary info seems ok on mobile, but very small on a desktop browser.
<img width="356" alt="Bildschirmfoto 2024-12-22 um 18 37 51" src="https://github.com/user-attachments/assets/9f6d18c5-9694-43de-a326-59f049784171" />
### Device Info
Firefox 133.0.3
MacOS 15.1.1

**Comment by selvalt7** [comment]
It seems that it's caused by how different is Firefox from the other browsers.

**Comment by Xlinx64** [comment]
In general it would be nice for the gauge to be more space efficient.
e.g:
- Remove the space under the gauge
- Make the labels in the gauge bigger
- Better padding/margin of the title

**Comment by selvalt7** [comment]
I tried adjusting the values a bit and it looks like this right now:

Desktop:
![desktop](https://github.com/user-attachments/assets/f3a3d75f-4569-465f-9d77-f6ed09aa419c)

Mobile:
![mobile](https://github.com/user-attachments/assets/011b3d0a-e8c7-41f4-bf3a-c1b1dc3b4b01)

The thing about the unused space on mobile is that gauge always have 1:1 aspect ratio and sections have variable columns width based on the viewport width.

So I think that it's better to keep gauge centered instead of stretching it.

I'm still thinking what to do with the header title, for now I've made it smaller and removed wrapping in favour of showing dots at the end when it's overflowing.
